src/libs/actions/Session.js

```diff
@@ -71,6 +71,33 @@ function signOut() {
         .catch(error => Onyx.merge(ONYXKEYS.SESSION, {error: error.message}));
 }
 
+/**
+ * Reopen the account and send the user a link to set password
+ *
+ * @param {String} [login]
+ */
+function reopenAccount(login = credentials.login) {
```

Comment:
NAB; Why are we defaulting `login` to `credentials.login` here and in `resendValidationLink`? Rephrased: When / why would a passed-in `login` string be different from what's stored in `credentials.login`?

I believe this has to do with unvalidated / closed accounts (the topic of your PR), but I think it would be useful to make this clear in the function comment

Reply:
So when you enter an email it hits this flow - https://github.com/Expensify/Expensify.cash/pull/2745/files#diff-283c48b7349b0bf7aba66da8891bc8a9f9df70664bd06e7a875f56b19c6c02f9R126 and credentials.login is not set yet.

But when you click resent link, it hits this flow - https://github.com/Expensify/Expensify.cash/pull/2745/files#diff-c075eec38f8adc854c238b1d4cfeba1ed3e5de0aa740aa78a658ffe3a53814f9R62 and does not need to pass an email since its in credentials.login. Does that make sense?